src/pages/index.vue

```diff
@@ -1,1643 +1,640 @@
-              src="~assets/images/index/grow-your-open-content.png"
-            />
-          </template>
-          <template #append>
-            <ButtonV2
-              :text="$t('index_page_new_culture_section_1_cta_button')"
-              theme="glow"
-              preset="tertiary"
-              :to="localeLocation({ name: 'about-writing-nft' })"
-              @click.native="
-                handleAboutWritingNFTButtonClickInNewCultureSection
-              "
-            />
-          </template>
-        </IndexPageHeadingContentWithCTA>
-
-        <IndexPageHeadingContentWithCTA
-          class="relative"
-          :heading="$t('index_page_new_culture_section_2_heading')"
-          :content="$t('index_page_new_culture_section_2_content')"
-        >
-          <template #image>
-            <img
-              class="laptop:hidden max-w-[276px]"
-              src="~assets/images/index/cultivate-readers-into-fans.png"
-            />
-          </template>
-          <template #append>
-            <ButtonV2
-              :text="$t('index_page_new_culture_section_2_cta_button')"
-              theme="glow"
-              preset="tertiary"
-              :to="
-                localeLocation({
-                  name: 'nft-class-classId',
-                  params: { classId: freeMintNFTClassId },
-                  query: { action: 'collect' },
-                })
-              "
-              @click.native="handleTryCollectButtonClickInNewCultureSection"
-            />
-          </template>
-        </IndexPageHeadingContentWithCTA>
-
-        <svg
-          class="hidden laptop:block absolute w-[680px] right-[1024px] bottom-0 translate-x-[100%] mr-[10%] blur-[10px]"
-          xmlns="http://www.w3.org/2000/svg"
-          xmlns:xlink="http://www.w3.org/1999/xlink"
-          viewBox="0 0 680 660"
-        >
-          <defs>
-            <linearGradient
-              id="tech-fill-gradient"
-              x1="1084.22"
-              y1="-713.01"
-              x2="734.34"
-              y2="-920.07"
-              gradientTransform="translate(-96.79 -831.81) rotate(30) scale(1 -1)"
-              gradientUnits="userSpaceOnUse"
+              <img
+                class="w-[100px] h-[100px]"
+                src="~assets/images/about/nft-book/icons/ownership.png"
+              />
+              <h4 class="font-600 mt-[3rem]">
+                {{ $t('about_nft_book_section_features_ownership_title') }}
+              </h4>
+              <p class="mt-[0.5rem] text-center">
+                {{
+                  $t('about_nft_book_section_features_ownership_description')
+                }}
+              </p>
+            </a>
+          </li>
+          <li>
+            <a
+              class="flex flex-col items-center"
+              href="https://blog.like.co/zh/%E5%85%A7%E5%AE%B9%E6%84%9B%E5%A5%BD%E8%80%85%E7%9A%84%E5%BB%A3%E5%A0%B4-liker-land-%E6%96%B0%E5%8A%9F%E8%83%BD/"
```

Comment:
Should this be locale sensitive? en version is https://blog.like.co/en/liker-land-the-plaza-for-content-lovers/